101-traffic-manager-webapp/README.md

```diff
@@ -0,0 +1,16 @@
+# Azure Traffic Manager with web apps
```

Comment:
Same case as for the VMs, it contains a few extra bits because we have to create the webapp but it's still a simple scenario for us.